internal/cmd/cliopts/flags_test.go

```diff
@@ -0,0 +1,208 @@
+package cliopts
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/spf13/pflag"
+	"gotest.tools/v3/assert"
+)
+
+func TestDefaultsFromEnv(t *testing.T) {
+	type target struct {
+		OneMore string
+		Next    int32
+		Many    []string
+	}
+
+	setup := func(tg *target) *pflag.FlagSet {
+		flags := pflag.NewFlagSet("testing", pflag.ContinueOnError)
+		flags.String("word", "default-value", "")
+		flags.Int("count", 12, "")
+		flags.String("two-parts", "", "")
+		flags.StringVar(&tg.OneMore, "one-more-extra-long", "", "")
+		flags.Int32Var(&tg.Next, "next", 0, "")
```

Comment:
I'll update this to use a non-zero default, to cover both the "bind to variable", and "lookup flag value" cases with default values.